### ISSUE TITLE
is this repo still accepting PRs for this season of hacktoberfest??

### ISSUE THREAD

**Issue by xenowits** [EricAlcaide/Rosetta_Stone#141]

**Comment by EricAlcaide** [comment]
i'll try, but try to create a new algorithm (new folder, readme, ....) that illustrates how to do funny things.
A few ideas:
* useful algorithms: dychotomous search, breadth-first search, depth-first search... 
* parallel programming
* asynchronous processes
* efficient information storage (hash tables, arrays, sets, ...)

**Comment by EricAlcaide** [comment]
i've got many things going on so maybe it takes 2-3 days to merge a PR... i'll try to do my best

**Comment by xenowits** [comment]
Yeah that's great @EricAlcaide !! I will try to push some cool algorithm implementations in go/js. Hope u review it😄
